src/com/ucm/dasi/catan/board/exception/InvalidBoardDimensionsException.java

```diff
@@ -0,0 +1,18 @@
+package com.ucm.dasi.catan.board.exception;
+
+public class InvalidBoardDimensionsException extends Exception {
+
+	/**
+	 * 
+	 */
+	private static final long serialVersionUID = -7651089991744581056L;
+	
+	public InvalidBoardDimensionsException() {
+		super();
+	}
+	
+	public InvalidBoardDimensionsException(String message) {
+		super(message);
```

Comment:
It is advisable to have an exception for each unexpected event. This usually creates confusion.

Reply:
Let's try not to use the term "event" in this context please.

I think that's a bit ambiguous. Why do you think the same exception is being used for different purposes?

Reply:
You are right, when I was referring to an event, I wanted to refer to something happening that did not follow the normal flow of execution, it seemed easier to call it an event, by shortening xD.

The problem of doing so is that, on the one hand, if it is necessary to refactor that exception in the future, for whatever reason, it becomes something much more tedious than if the standard message will be implemented directly in the class. In addition, in this way, certain information is lost as to why the exception occurred. That is, the only thing that the difference is the message and not the class itself. Now that there are few exceptions, it may not be a problem, but if one exception will trigger another, we would begin to have trouble finding out where the exception arose.